src/faction_camp.cpp

```diff
@@ -346,6 +346,103 @@ static std::string camp_trip_description( const time_duration &total_time,
  */
 static bool survive_random_encounter( npc &comp, std::string &situation, int favor, int threat );
 
+static std::map<risk_diff_level, std::string> risk_diff_to_risk_name( {
```

Comment:
Instead of having these output a single word, have them output the whole string,  i.e. "Risk: low".
Chained substitutions like this make the strings harder to translate.

Reply:
One of my goals here was to reduce the number of times a translator needs to translate the labels like "Risk", "Difficulty", "Effects", etc.
Also, I anticipate this bare string getting used in a nicer UI later, where the label is not part of the same text field, such as a table view where "Risk" is a column.

Reply:
Do you mean "Risk" in a different part of the codebase where it might be used differently? I think the fix for that is to provide a translation context, e.g. `translation( "faction mission activity", "Risk: %s" )`.

Reply:
Translation contexts are only a partial solution.

Consider: "Risk: very high" vs. "Effect: increase risk level to very high" vs "Effect: very high chance of increased risk"

In 1, risk is nominative, very high is nominative. Fine and dandy
2, risk is genitive, very high is genitive (don't quote me on that, I'm having trouble equating my native case names and English case names)
3. very high is nominative but different ending because the word "chance" is feminine, risk is dative (uh? I think? see point 2 above

So you can't reuse ANYTHING across those three seemingly simple phrases, and I can't see a way you could disambiguate 2 and 3 using contexts either

Reply:
Here's an existing place where we translate single words with translation context, for a similar use case: https://github.com/CleverRaven/Cataclysm-DDA/blob/950e07ee27a9d52c2f90fe22dcd6a17304b56f5f/src/display.cpp#L525-L527

Reply:
This isn't optional, feel free to ask about aspects of this that are unclear to you, but emitting the whole string whenever possible is far more important than "reducing redundancy".

Reply:
It is unclear to me why we can isolate "Light" and "Moderate" exercise levels, "Fresh" and "Very" weariness levels, etc, but "Low" and "Medium" risk levels somehow don't fit that pattern.

Reply:
I'm not even sure how such a change could even work, given the sidebar widgets for those strings. There are JSON-defined uses of those strings, which would not at all be amenable to combining the labels with each string.

I'll end up combining them here to get this merged, but I don't see a path forward that doesn't involve re-splitting them as soon as they are going to get used in another context.

Reply:
To be clear, I did make the requested change, so the relevant strings now look like `_( "Risk: Low" )`, `_( "Danger: High" )`, etc.